### DISCUSSION TITLE
Can not find code associated with the course.

### DISCUSSION THREAD

**Discussion by bracketloader** [Cyfrin/foundry-full-course-cu#584] [Q&A] [answered]
I am really sorry but can someone tell me where the code files are I am unable to find them.

**Comment by shikhar229169** [comment, marked as answer]
Go to the below link:
```
https://github.com/Cyfrin/foundry-full-course-f23#table-of-contents
```
Over there you will find links to all lesson, click on the link you want the code for, and then click on the link under label code.

You can also find the code to individual repo here:
```
https://github.com/orgs/Cyfrin/repositories
```

**Comment by DevSwayam** [comment]
> I am really sorry but can someone tell me where the code files are I am unable to find them.

Sir if you got your answer can you mark it as answred.

**Comment by bracketloader** [comment]
thank you;
